images/base/Dockerfile

```diff
@@ -76,14 +76,16 @@ RUN curl -fsSL "https://download.docker.com/linux/$(. /etc/os-release; echo "$ID
     && clean-install "docker-ce=${DOCKER_VERSION}"
 
 # Enable CRI for containerd
-RUN mkdir -p /etc/docker && echo '{"cri-containerd": true}' > /etc/docker/daemon.json
-RUN mkdir -p /etc/containerd && containerd config default > /etc/containerd/config.toml
+RUN if [ -x "$(command -v containerd)" ]; then \
+  mkdir -p /etc/docker && echo '{"cri-containerd": true}' > /etc/docker/daemon.json;\
+  mkdir -p /etc/containerd && containerd config default > /etc/containerd/config.toml;\
+fi
 
 # Install CNI binaries to /opt/cni/bin
 # TODO(bentheelder): doc why / what here
 ARG CNI_VERSION="0.7.5"
 ARG CNI_BASE_URL="https://storage.googleapis.com/kubernetes-release/network-plugins/"
-RUN export ARCH=$(dpkg --print-architecture) \
+RUN export ARCH=$(dpkg --print-architecture| sed 's/ppc64el/ppc64le/') \
```

Comment:
oh that is wildly irritating. can we get that fixed??

Reply:
there is been already concerned raised about this and here is the explanation given - https://lists.debian.org/debian-powerpc/2014/08/msg00042.html, we don’t have any other go, will have to live with this as of now.